representation/update_representations/src/index.ts

```diff
@@ -0,0 +1,86 @@
+import { createRootViewer } from "./common/init";
+import { StateSelection, StateTransform } from "molstar/lib/mol-state";
+import { createStructureRepresentationParams } from "molstar/lib/mol-plugin-state/helpers/structure-representation-params";
+
+const byres = document.getElementById("byres")!;
+const bychain = document.getElementById("bychain")!;
+const bypos = document.getElementById("bypos")!;
+
+async function init() {
+  // Create Viewer. Returns a PluginContext
+  const plugin = await createRootViewer();
+
+  // Download data as mmCIF
+  const fileData = await plugin.builders.data.download({
+    url: "https://models.rcsb.org/5ee7.bcif",
+    isBinary: true,
+  });
+
+  // Load mmCIF and create trajectory -> model -> structure -> representation
```

Comment:
Would be helpful to explain the difference between trajectory -> model -> structure -> representation

Reply:
@frederickparsons-tandemai in each example? This was detailed in the create_representation example